tox.ini

```diff
@@ -23,6 +25,8 @@ commands =
     sh -c "SITE_URL=https://treeherder.dev TREEHERDER_DEBUG=False ./manage.py check --deploy --fail-level WARNING"
     # Exercising running tests outside of Docker
     pytest tests/ --ignore=tests/selenium --ignore=tests/extract
+    # This is to deal with running the containers with --detached
+    docker-compose down
```

Comment:
I believe it is better to use with [commands_post](https://tox.readthedocs.io/en/latest/config.html#conf-commands_post)